packages/@webex/plugin-authorization-browser-first-party/src/authorization.js

```diff
@@ -220,10 +220,34 @@ const Authorization = WebexPlugin.extend({
    */
   initiateAuthorizationCodeGrant(options) {
     this.logger.info('authorization: initiating authorization code grant flow');
-    this.webex.getWindow().location = this.webex.credentials.buildLoginUrl(
+    const loginUrl = this.webex.credentials.buildLoginUrl(
       Object.assign({response_type: 'code'}, options)
     );
 
+    if (options?.separateWindow) {
+      // Default window settings
+      const defaultWindowSettings = {
+      width: 600,
+      height: 800
+      };
+
+      // Merge user provided settings with defaults
+      const windowSettings = Object.assign(
+      defaultWindowSettings, 
+      typeof options.separateWindow === 'object' ? options.separateWindow : {}
+      );
+      // Convert settings object to window.open features string
+      const windowFeatures = Object.entries(windowSettings)
+      .map(([key, value]) => `${key}=${value}`)
+      .join(',');
+      this.webex.getWindow().open(loginUrl, '_blank', windowFeatures);
+    } else {
+      // Default behavior - open in same window
+      this.webex.getWindow().location = loginUrl;
+    }
```

Comment:
Why is the indentation weird throughout this PR?  It makes it hard to follow the code at a glance.